book/website/reproducible-research/renv/renv-containers.md

```diff
@@ -395,22 +395,39 @@ Once the researcher is done, they can close the container as normal.
 When they come back to the project and want to continue their work, they only need to use the same command as above, and it will load the work contained in `volume_name` into the new container.
 It will save any new work there too.
 
-Below is a list of volume related commands:
+Below is a list of volume related commands: 
 
 - To list volumes: `sudo docker volume ls`
 - To delete a volume: `sudo docker volume rm volume_name`
 - To delete all unattached volumes: `sudo docker volume prune`
 
 If, when deleting a container, a `-v` is included after `rm` in `sudo docker rm container_ID`, any volumes associated with the container will also be deleted.
 
+(rr-renv-containers-rootless)=
+## Docker without root access
+
+Up until April 2020, the only way to run Docker was with root access. 
+"Rootless" mode was made available as part of the [v20.10](https://docs.docker.com/engine/security/rootless/) release.
+ Rootless mode is currently only avaliable on Linux and requires an initial install of Docker >= v20.10. 
+ 
+ The underyling difference between Docker and rootless mode is that perviously any system running docker had a daemon running as `uid0` that creates and owns all images, but with rootless mode the user creates and owns any images that they initialize.
+ To install and run the rootless version of Docker as a non-root user, use the following commands:
+
+```
+dockerd-rootless-setuptool.sh install
+docker run -d --name dind-rootless --privileged docker:20.10-dind-rootless
```

Comment:
Does `20.10` refer to the installed version of Docker?

Reply:
that's correct!